code/core/fastTrack.py

```diff
@@ -27,6 +26,10 @@ def __init__(self, handler):
 
     def is_fastTrack_eligible(self):
         """Check if query is eligible for fast track processing"""
+        # Skip fast track for sites without embeddings
+        if self.handler.site.lower() == "datacommons":
+            logger.debug("Fast track not eligible: DataCommons site has no embeddings")
+            return False
         if (self.handler.context_url != ''):
```

Comment:
Is there a more generic way to code this?  Like a no embeddings flag?  Is there a way to detect this programmatically instead of hardcoding site names?